src/functions-reference/bounded_discrete_distributions.Rmd

```diff
@@ -155,6 +155,132 @@ The log binomial probability mass of n successes in N trials given
 logit-scaled chance of success alpha dropping constant additive terms
 `r since("2.25")`
 
+## Binomial-logit generalized linear model (Logistic Regression) {#binomial-logit-glm}
+
+Stan also supplies a single function for a generalized linear model
+with Binomial likelihood and logit link function, i.e. a function
+for logistic regression with aggregated outcomes. This provides a more efficient
+implementation of logistic regression than a manually written
+regression in terms of a Binomial likelihood and matrix
+multiplication.
+
+### Probability mass function
+
+Suppose $N \in \mathbb{N}$, $x\in \mathbb{R}^{n\cdot m}, \alpha \in \mathbb{R}^n, \beta \in \mathbb{R}^m$, and $n \in
```

Comment:
I would start by saying that M is the number of predictors and N is the number of data items.

Then we need the notation to match with caps and to use \times not \cdot

* x is in R^(M x N) (with \times, not \cdot)
* \alpha in \mathbb{R}
* \beta \in \mathbb{R}^M

Reply:
> Then we need the notation to match with caps and to use \times not \cdot

That's also inconsistent with the other `_glm` docs:

  - [`normal_id_glm`](https://github.com/stan-dev/docs/blob/0398ca443f4ad9bf98560a50ac0f6615b0438c84/src/functions-reference/unbounded_continuous_distributions.Rmd#L216)
  - [`bernoulli_logit_glm`](https://github.com/stan-dev/docs/blob/0398ca443f4ad9bf98560a50ac0f6615b0438c84/src/functions-reference/binary_distributions.Rmd#L156)